allvars.h

```diff
@@ -197,7 +197,11 @@ extern struct global_data_all_processes
 
     int MaxPart;			/*!< This gives the maxmimum number of particles that can be stored on one
                               processor. */
-    int MaxPartBh;		/*!< This gives the maxmimum number of BH particles that can be stored on one
+    int MaxPartSph;		/*!< This gives the maxmimum number of Sph particles that can be stored on this
```

Comment:
Use an array?

Reply:
It is possible, but I am not so sure it is cleaner, honestly. We use bare variables for the slots as well...